src/components/side_panel/data_validation/dv_editor/dv_editor.ts

```diff
@@ -60,6 +65,7 @@ export class DataValidationEditor extends Component<Props, SpreadsheetChildEnv>
 
   onCriterionTypeChanged(type: DataValidationCriterionType) {
     this.state.rule.criterion.type = type;
+    this.state.isTypeUpdated = true;
```

Comment:
we should probably have the same behaviour for CFs and in the data filter menu, no ? To confirm with frgi, but I don't see why implement it in DVs but not CFs

Reply:
It's done :wink: